app/models/user.rb

```diff
@@ -2,14 +2,15 @@ class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :validatable, :confirmable
+         :recoverable, :rememberable, :validatable # :confirmable
   has_many :posts, foreign_key: 'author_id', dependent: :destroy
   has_many :comments
-  validates :name, presence: true
+  # validates :name, presence: true
   validates :photo, presence: true
```

Comment:
- [ ] [OPTIONAL] You have a working project, however, kindly remove this validation as it prevents new users from signing up.
<img width="1197" alt="Screenshot 2023-06-23 at 22 56 37" src="https://github.com/monicah79/Renters_ke/assets/58692590/7a39c9c7-96e5-4a43-a3d6-2b98b0588c21">